docs/header.sticky.md

```diff
@@ -1,12 +1,35 @@
 # qgrid 
+
+## Option 1 - settings in template
```

Comment:
As for me, the second options looks more preferable, because sticky is about look-and-feel only, there's no related logic from user's point of view.